.github/workflows/release-images.yaml

```diff
@@ -111,6 +147,16 @@ jobs:
             IMAGE_NAMESPACE=${{ github.repository }} \
             PLATFORM=${{ matrix.platform }}/${{ matrix.arch }} \
             BUILDX_ACTION=--push
+          
+      - name: Sign container image
+        run: |
+          for image in \
+            ghcr.io/${{ github.repository }}/retina-operator:$TAG \
+            ghcr.io/${{ github.repository }}/retina-operator:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
+          ; do
+            DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ${image})
+            cosign sign --yes ${DIGEST}
+          done
 
   manifests:
```

Comment:
Will we need to sign the multiplat manifests during this job also?

Reply:
Well yes, that's where the images without `${os}-${arch}` got generated and pushed. Thanks for the callout!

Reply:
This is added in HEAD